### PR TITLE
Provide run number to infologger

### PR DESCRIPTION
**Pull request by ktf**

### CODE REVIEW

**Comment by ktf** [comment]
@TimoWilken how is it possible that there are still 5 PRs in front of this one? There used to be 2 yesterday.

File: Framework/Core/include/Framework/ServiceSpec.h [hunk 1]
```diff
@@ -162,6 +167,11 @@ struct ServiceDispatchingHandle {
   void* service;
 };
 
+struct ServiceStartHandle {
+  ServiceExitCallback callback;
```
Comment (wiechula) [inline Framework/Core/include/Framework/ServiceSpec.h]: Should it be
```suggestion
  ServiceStartCallback callback;
```

**Comment by TimoWilken** [comment]
@ktf I marked the fullCI builds yellow yesterday afternoon after a fix to the o2checkcode build script was merged in alidist.

**Comment by ktf** [comment]
@wiechula thanks. I guess it compiles / works fine because it's the same type underneath.

**Pull request merged**